### ISSUE TITLE
Debug hover doesn't work with spread operator

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#48973]
Debug code like

```ts
arr.push(...veryLargeArray);
```

- Hover `veryLargeArray`, nothing shows up.
- Other hovers work fine.

**Comment by isidorn** [comment]
Yes the issue is in our expression matching regex [here](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/debug/electron-browser/debugHover.ts#L147)
For example we have to support `rob.name` with that regex. Feel free to update that regex to support this case, hope it does not overcompllicate it too much.
